docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerCloud.java

```diff
@@ -254,7 +249,8 @@ public Node call() throws Exception {
                                 // TODO: record the output somewhere
                                 DockerSlave slave = null;
                                 try {
-                                    slave = t.provision(new StreamTaskListener(System.out));
+//                                    slave = t.provision(new StreamTaskListener(System.out));
+                                    slave = provision(t);
```

Comment:
This part of code is not mine and of course it weird for me, my idea was to implement all objects and pass to Cloud API .